MAUI/DataGrid/Styling.md

```diff
@@ -0,0 +1,435 @@
+You can customize the alternate row count for applying the alternate row style using the [SfDataGrid.AlternationRowCount] (https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.SfDataGrid.html#Syncfusion_Maui_DataGrid_SfDataGrid_AlternationRowCountProperty) property.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml"%}
+<ContentPage xmlns:syncfusion="http://schemas.syncfusion.com/maui">
+    <ContentPage.Content>
+        <syncfusion:SfDataGrid AlternationRowCount="3" 
+                            ItemsSource="{Binding OrderInfoCollection}">
+            <syncfusion:SfDataGrid.DefaultStyle>
+                <syncfusion:DataGridStyle AlternateRowBackground="Bisque"/>
+            </syncfusion:SfDataGrid.DefaultStyle>
+        </syncfusion:SfDataGrid>
+    </ContentPage.Content>
+</ContentPage>
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+public partial class MainPage : ContentPage
+{
+    public MainPage()
+    {
+        InitializeComponent();
+        this.dataGrid.DefaultStyle.AlternateRowBackground = Colors.Bisque;
+        this.dataGrid.AlternationRowCount = 3;
+    }
+}
+{% endhighlight %}
+{% endtabs %}
+
+![Changing the alternation row count in .NET MAUI DataGrid](Images/styling/maui-datagrid-Alternation-rowcount.png)
+
+## Changing the font style
+You can apply the style for header and row font attributes by using SfDataGrid.DefaultStyle.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml"%}
+<ContentPage xmlns:syncfusion="http://schemas.syncfusion.com/maui">
+    <ContentPage.Content>
+        <syncfusion:SfDataGrid ItemsSource="{Binding OrderInfoCollection}">
+            <syncfusion:SfDataGrid.DefaultStyle>
+                <syncfusion:DataGridStyle HeaderRowFontAttributes="Bold"
+                                        HeaderRowFontFamily="TimesNewRoman"
+                                        HeaderRowFontSize="16"
+                                        RowFontAttributes="Italic"
+                                        RowFontFamily="Adabi"
+                                        RowFontSize="14"/>
+            </syncfusion:SfDataGrid.DefaultStyle>
+        </syncfusion:SfDataGrid>
+    </ContentPage.Content>
+</ContentPage>
+{% endhighlight %}
+{% endtabs %}
+
+![Changing the font style .NET MAUI DataGrid](Images/styling/maui-datagrid-font-style.png)
+
+## Border customization
+The data grid customizes the grid borders to vertical, horizontal, both or none. Set desired value to [SfDataGrid.GridLinesVisibility] (https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.SfDataGrid.html#Syncfusion_Maui_DataGrid_SfDataGrid_GridLinesVisibilityProperty) or [SfDataGrid.HeaderGridLinesVisibility] (https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.SfDataGrid.html#Syncfusion_Maui_DataGrid_SfDataGrid_HeaderGridLinesVisibilityProperty)
+Following are the list of options available to customize the grid borders:
+.	Both
+.	Horizontal
+.	Vertical
+.	None
+
+### Both
```

Comment:
@FarjanaParveen  - I think no need to document all the border types. Just give the code snippet for both setting for data row and header row. Its enough.